src/main/kotlin/nl/info/zac/zaak/ZaakService.kt

```diff
@@ -151,6 +153,18 @@ class ZaakService @Inject constructor(
         }
     }
 
+    fun retrieveZaakAndZaakType(zaakURI: URI): Pair<Zaak, ZaakType> =
```

Comment:
How are these two functions used? I cannot find a direct invocation

Reply:
It's a large diff so you need to open it first. :-)